.github/workflows/release.yml

```diff
@@ -11,13 +11,14 @@ on:
 
 permissions:
   contents: write
+  id-token: write
```

Comment:
I haven't found docs on why this is needed, is there any docs we can add here?

Reply:
@psmulovics The second link - https://docs.npmjs.com/generating-provenance-statements